pulp_ansible/app/tasks/collections.py

```diff
@@ -431,7 +431,17 @@ def _update_highest_version(collection_version):
     Otherwise does nothing.
     """
 
+    def is_higher(a, b):
+        """Is a > b ?"""
+        if a.prerelease and b.prerelease and a > b:
+            return True
+        if a.prerelease and not b.prerelease:
+            return False
```

Comment:
Don't you also need to account for the scenario when `a` is not a prerelease, but `b` is?